test/verify/check-apps

```diff
@@ -52,6 +52,10 @@ class TestApps(PackageCase):
     <name>{0}</name>
     <summary>An application for testing</summary>
     <launchable type="cockpit-manifest">{0}</launchable>
+    <description>
+        <p>DESCRIPTION:none</p>
```

Comment:
@mvollmer, Is this required in the app-data spec? Should we be adding this here? Or should we make the code more robust so that it doesn't error if this field is missing?

Reply:
`<description>` is optional.  We should fix the code.

Reply:
Oh, this is already done and merged... :-)